tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py

```diff
@@ -58,42 +59,76 @@ def check_forward(self, x_data, t_data, use_cudnn=True):
                 loss_expect /= self.t.shape[0]
         self.assertAlmostEqual(loss_expect, loss_value, places=5)
 
+    def check_forward_without_reduction(self, x_data, t_data, use_cudnn=True):
+        x_val = chainer.Variable(x_data)
+        t_val = chainer.Variable(t_data)
+        loss = functions.sigmoid_cross_entropy(
+            x_val, t_val, use_cudnn, self.normalize, reduce=False)
+        self.assertEqual(loss.data.shape, self.x.shape)
+        self.assertEqual(loss.data.dtype, numpy.float32)
+        loss_value = cuda.to_cpu(loss.data)
+
+        # Compute expected value
+        if not getattr(self, 'ignore_all', False):
```

Comment:
We have to check the output even if `ignore_all` is `True`. So could you remove this line?